### PR TITLE
chore: fix some comments

### PR DESCRIPTION
**Pull request by beforetech**
fix some comments

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Review by jennifer-shehane — APPROVED** [review]

**Comment by jennifer-shehane** [comment]
@beforetech Thanks for the contribution! Could you please sign our CLA?

File: packages/data-context/test/unit/sources/FileDataSource.spec.ts [hunk 1]
```diff
@@ -72,7 +72,7 @@ describe('FileDataSource', () => {
           await fs.mkdirs(nestedScriptPath)
           await fs.writeFile(path.join(nestedScriptPath, 'nested-script.js'), '')
 
-          // Verify that the glob pattern is not impacted if if contains directories equivalent
+          // Verify that the glob pattern is not impacted if contains directories equivalent
```
Comment (MikeMcC399) [inline packages/data-context/test/unit/sources/FileDataSource.spec.ts]:
```suggestion
          // Verify that the glob pattern is not impacted if it contains directories equivalent
```

**Comment by jennifer-shehane** [comment]
@beforetech We'll be closing this PR without a CLA signature.

**Comment by beforetech** [comment]
> fix some comments

Thanks. The new pr https://github.com/cypress-io/cypress/pull/30053

Please review again.

**Pull request closed without merging**